hazelcast/server/src/jepsen/hazelcast_server.clj

```diff
@@ -5,15 +5,42 @@
             [clojure.java.io :as io]
             [clojure.string :as str])
   (:import (com.hazelcast.core Hazelcast)
+           (com.hazelcast.config.cp FencedLockConfig CPSemaphoreConfig)
            (com.hazelcast.config Config
                                  LockConfig
                                  MapConfig
-                                 QuorumConfig)))
+                                 QuorumConfig)
+           ))
 
 (def opt-spec
   [["-m" "--members MEMBER-LIST" "Comma-separated list of peers to connect to"
     :parse-fn (fn [m]
-                (str/split m #"\s*,\s*"))]])
+                  (str/split m #"\s*,\s*"))]])
+
+(defn prepareCPSubsystemConfig
+  "Prepare Hazelcast CPSubsystemConfig"
+  [config members]
+  (let [cpSubsystemConfig (.getCPSubsystemConfig config)
+        raftAlgorithmConfig (.getRaftAlgorithmConfig cpSubsystemConfig)
+        semaphoreConfig (CPSemaphoreConfig. "jepsen.cpSemaphore" false)
+        lockConfig1 (FencedLockConfig. "jepsen.cpLock1" 1)
+        lockConfig2 (FencedLockConfig. "jepsen.cpLock2" 2)
+
+        _ (.setLeaderElectionTimeoutInMillis raftAlgorithmConfig 1000)
```

Comment:
Since you're not using any of these return values, and there's no bindings after them, you can move all of these expressions into the body of the `let` expression, and drop the `_`s.